src/instantiator.js

```diff
@@ -105,6 +105,13 @@ function instantiate(schema, options) {
       return;
     }
     var i;
+    // Check if type is array of types
```

Comment:
Please change the comments to the conventions of this project:

1.  Begin with a capitalized letter.
2.  End with a dot.